apps/tasks/lib/screens/main_screen_subscreens/my_tasks_screen.dart

```diff
@@ -10,6 +10,7 @@ import 'package:tasks/dataclasses/subtask.dart';
 import 'package:tasks/screens/settings_screen.dart';
 import 'package:helpwave_widget/loading.dart';
 
+import '../../components/task_expansion_tile.dart';
```

Comment:
Remove the empty line
```suggestion
import 'package:helpwave_widget/loading.dart';
import '../../components/task_expansion_tile.dart';
```